### PR TITLE
Finish Project

### PR DESCRIPTION
**Pull request by yayoamigo**
In this pull request I was able to :
-Get metric Data from API.
-Format Data in thunk to be easily used by the app.
-Create dynamic route that changes for each continent clicked.
-Implement searching tool to help users find their target easily.
-Add transitions and styling to meet the required template.
-Add tests .
-Change README to describe the project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,161 @@
...
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please give it a star 😁🌟✨
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+We would like to thank Microverse for the project UI.
+
```
Comment (Ismailco) [inline README.md]: - [x] The README should contain an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/), kindly make sure to give credit to Nelson Sakwa

File: src/Tests/Nav.test.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+test('renders content', () => {
+  const component = render(
+    <Router>
+      <Provider store={store}>
+        <Nav />
+      </Provider>
+    </Router>,
+  );
+  expect(component.container).toHaveTextContent('Air polution app');
```
Comment (Ismailco) [inline src/Tests/Nav.test.js:30]: - [x] This Test is failing, kindly make sure to fix it so it passes, here is the error message produced 👇🏼

Screenshot_20230106_223819

**Review by Ismailco — CHANGES_REQUESTED** [review]
#### Hi @yayoamigo, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
You did a great job, keep up the good work 👍🏼

giphy


## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Suggested changes

**I highly suggest using this PR link to request a review and send the project after the review**

_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/App.css [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+}
+
+button:active {
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+  transform: translateY(2px);
+}
+
+.selectdiv {
+  display: inline-block;
+  position: relative;
```
Comment (MeqdamAlqudah) [inline src/App.css:128]: - [Optional ] Please make sure to fix the errors that appear in the console, I think the first one is about the map please try to work on this and understand the map data better, the second one is about the prop type also work on it please.
image

- [ Optional]  Please try to change the items sizes to match the design 
image

File: src/App.css [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+  font-size: 16px;
+  color: #3498db;
```
Comment (MeqdamAlqudah) [inline src/App.css:130]: - [ Optional] Please use the same color as in the design for items
image

**Review by MeqdamAlqudah — APPROVED** [review]
Hi @yayoamigo , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by yayoamigo** [comment]
Just did, thanks for the feedback. I don't know why but it doesn't appear to be approved in my dashboard @MeqdamAlqudah

**Comment by zieeco** [comment]
@yayoamigo, Your dashboard would be updated soon.

**Comment by zieeco** [comment]
@yayoamigo, What does it show on your dashboard?

**Comment by yayoamigo** [comment]
> @yayoamigo, What does it show on your dashboard?

rr

**Review by zieeco — COMMENTED** [review]
Hi @yayoamigo ,

 Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: Congratulations! tada
 
 
 Cheers and Happy coding!clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by zieeco** [comment]
@yayoamigo, kindly check your dashboard now and let me know if it has changed with a screenshot of it as you did above. :pray:

**Comment by yayoamigo** [comment]
> @yayoamigo, kindly check your dashboard now and let me know if it has changed with a screenshot of it as you did above. :pray: 

It works:) thanks

**Comment by zieeco** [comment]
> > @yayoamigo, kindly check your dashboard now and let me know if it has changed with a screenshot of it as you did above. pray
> 
> It works:) thanks

Please, add the screenshot of it

**Pull request merged**